gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/zosmf/ZosmfBeans.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.gateway.security.service.zosmf;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+
+/**
+ * Config for zOSMF beans
+ */
+@Configuration
+public class ZosmfBeans {
+
+    @Bean

```

Comment:
Wouldn't be necessary to use `@Qualifier` on the bean in case of future different implementations? Just asking, it can be added once there will be a new one of course

Reply:
The idea is that there should be just one at any time. The others would be conditioned out by Spring annotations

Reply:
that way it would not pose a problem. Spring will refuse to start if there are multiple candidates for wiring, so it's safe

Reply:
ok thanks for expaining. Sounds good.